client/js/config/styles.js

```diff
@@ -1 +1,182 @@
-// for shared styles (typography, colours, etc.)
+import { Dimensions } from 'react-native';
+
+export const dimensions = {
+  fullHeight: Dimensions.get('window').height,
+  fullWidth: Dimensions.get('window').width
+};
+
+export const colors = {
+  neutralLight: '#dddddd', // light grey
+  neutralDark: '#636363', // med grey
+  brand: '#eea922', // gold
+  white: '#ffffff',
+  black: '#000000'
+};
+
+export const center = { justifyContent: 'center', alignItems: 'center' };
+
+export const row = { flexDirection: 'row' };
+
+export const jusCenterRow = {
+  ...row,
+  justifyContent: 'center'
+};
+
+export const iteCenterRow = {
+  ...row,
+  alignItems: 'center'
+};
+
+export const margin = {
+  sm: 10,
+  md: 20,
+  lg: 30,
+  xl: 40
+};
+
+export const padding = {
+  xxxs: 2,
+  xxs: 6,
+  xs: 10,
+  sm: 15,
+  md: 20,
+  lg: 30,
+  xl: 40
+};
+
+export const pb = {
+  paddingBottom: padding.xs
+};
+
+export const pb0 = {
+  paddingBottom: 0
+};
+
+export const pt0 = {
+  paddingTop: 0
+};
+
+export const pt1 = {
+  paddingTop: padding.lg
+};
```

Comment:
this creates an unnecessary graph dependency in your project. If you want to use padding top/bottom just put styles in your styles.js file instead of exporting one line style code.

Reply:
This has been removed and the change is in the jl-feature-custom-text branch.